### ISSUE TITLE
Crate CMR-publication UI/tool

### ISSUE THREAD

**Issue by rtapella** [MAAP-Project/ZenHub#662]
Jobs UI Usability Test

"Would like to point at a path of validated data or an index table, then publish the batch to CMR (after validating it)"

Rather than adding CMR publication up-front with the job-submit, make a separate tool that can be used to quickly publish validated data to CMR

**Comment by chuckwondo** [comment]
@rtapella for clarification, what do you mean by "validating it"? I'm asking because generating the data is insufficient for publication, as the CMR requires metadata, but in my limited experience with what users are doing in the MAAP, the users are not generated the corresponding metadata, or are they?

**Comment by rtapella** [comment]
By "validating" I mean testing it to make sure that the data is "correct" to some standard test process. If someone is generating data that they'd like to put into CMR then they'd have to make sure the data looks good and also generate/input the appropriate metadata.

**Comment by rtapella** [comment]
(The comment stems from testing the new Job Submit UI where there is an inactive switch to push outputs into CMR. That triggered a discussion that data would not typically be pushed into CMR without some sort of manual process in-between for testing. Hence the suggestion that we pull the CMR-publication part out of the job-submit process and into a separate tool.)

**Comment by chuckwondo** [comment]
Thanks for clarifying. I'd suggest that this issue perhaps need further clarification, because I see at least these distinct steps:

1. validate generated data
2. generate collection metadata, if data is for a new collection
3. publish collection metadata to the CMR, if newly created
4. generate granule metadata related to the generated data
5. publish granule metadata to the CMR

Further, I imagine we would want to first publish metadata to the UAT CMR for review, prior to publishing to the OPS CMR.

Since "publishing to CMR" is not really conducive to a single push-button action (i.e., the rough workflow above requires several steps, including reviews), this might not be something appropriate to surface as part of the Jobs UI.

**Comment by rtapella** [comment]
I agree, there should be a whole process behind this and "best practice".

**Comment by marjo-luc** [comment]
@chuckwondo  @rtapella Is this ticket still relevant?

**Comment by rtapella** [comment]
I think it’s still kind of relevant, but with STAC instead of CMR.
I am not sure that it’s a development ticket though… it’s more like a SOP/process to follow to get your data published in the MAAP STAC.
